### ISSUE TITLE
GOG games connect for steam not working

### ISSUE THREAD

**Issue by jerusiek** [FriendsOfGalaxy/galaxy-integration-steam#163]
So i connected all my other accounts with games and it worked well, but steam connects and says OFFLINE then after closing menu and opening it back its not connected at all.

[https://snipboard.io/DV7rlU.jpg](url)

Screenshot link^^^^^

**Comment by Rogh64** [comment]
Me too

**Comment by Victor1733** [comment]
Same here.

**Comment by jerusiek** [comment]
> Same here.

At least im not alone

**Comment by Regocenlo** [comment]
here is a discussion on the subject <https://github.com/FriendsOfGalaxy/galaxy-integration-steam/issues/159> I think they still can't fix it

**Comment by Meshy24** [comment]
Same here. I can't really pin point when it started though. It was either 2.0.63 or 2.0.62... possibly earlier. It might be something that broke on Steam's end too.

**Comment by Amarinth** [comment]
@Meshy24 It broke when Steam changed how their authentication process works. They made it more secure, but also a lot more complicated.  Simply put nothing about the way the original plugin handled keeping your Steam account signed in applies anymore, which is why it's been broken for so long, because there is no quick fix.

The good news is that @ABaumher has been working tirelessly to restore the plugin to something resembling a working order, and there's actually an [alpha build](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/discussions/169) up right now if anyone wants to help with testing it.

Also you can keep an eye on the feed over at [Issue #159](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/issues/159) for progress updates.

**Comment by Meshy24** [comment]
> 

Yeah, of course it's some security upgrade. And that plugin is working fantastic so far! All hail @ABaumher!

**Comment by Bansheedragon** [comment]
Is this issue being fixed?
I'm having the same issue every time I launch it or have to restart for an update and clicking retry does not work.

The workaround I found so far is to disconnect steam and then reconnnect it.
It means I have to input a steam Guard key every time, but at least it's working.

But then again I would rather not have to do that, I don't want to have a repeat of a situation I had with Ubisoft connect a while back where my Ubisoft account got a temporary suspension for trying to connect too many times, so it would be better if it just reconnected on it's own or I could click retry and have it work

**Comment by ABaumher** [comment]
#159

Tl;Dr: yes, but not here. The maintainer here is inactive. Try https://github.com/ABaumher/galaxy-integration-steam/releases/tag/v1.0.5-beta
